crates/cheatcodes/spec/src/vm.rs

```diff
@@ -2139,6 +2139,10 @@ interface Vm {
     /// Returns ENS namehash for provided string.
     #[cheatcode(group = Utilities)]
     function ensNamehash(string calldata name) external pure returns (bytes32);
+
+    /// Returns a random uint256 value.
+    #[cheatcode(group = Utilities)]
+    function randomUint() external returns (uint256);
```

Comment:
we also want an overloaded fn that accepts a range

and randomAddress perhaps?

Reply:
I don't know if range makes sense since you can `bound` it later.